modules/nf-core/checkm/qa/main.nf

```diff
@@ -23,8 +23,8 @@ process CHECKM_QA {
     def args     = task.ext.args ?: ''
     prefix       = task.ext.prefix ?: "${meta.id}"
     suffix       = task.ext.args?.matches(".*-o 9.*|.*--out_file 9.*") ? "fasta" : "txt"
-    def coverage = coverage_file                                   ? "--coverage_file ${coverage_file}"  : ""
-    def exclude  = exclude_marker_file                             ? "--exclude_markers ${marker_filer}" : ""
+    def coverage = coverage_file.isFile()                              ? "--coverage_file ${coverage_file}"  : ""
+    def exclude  = exclude_marker_file                                 ? "--exclude_markers ${exclude_marker_file}" : ""
```

Comment:
Would it make sense to check if `exclude_marker_file` is a valid file as you did for `coverage`?

Reply:
I left both the condition to  check if it's not empty and if it's file